src/applications/personalization/profile/tests/fixtures/communication-preferences/get-200-maximal.json

```diff
@@ -19,8 +19,13 @@
                   "description": "SMS Notification",
                   "communicationPermission": {
                     "id": 4281,
-                    "allowed": false
+                    "allowed": true
                   }
+                },
```

Comment:
just for testing purposes so we can see what the page looks like when an notification can be received by both text and email